### PR TITLE
[ONNX] Update len symbolic

### PR DESCRIPTION
**Pull request by KsenijaS**
Update len symbolic.

### CODE REVIEW

File: torch/onnx/symbolic_opset11.py [hunk 1]
```diff
@@ -273,7 +273,9 @@ def masked_scatter(g, self, mask, source):
 
 
 def _len(g, self):
-    return g.op("SequenceLength", self)
+    if self.type().isSubtypeOf(torch._C.ListType.ofTensors()):
```
Comment (neginraoof) [inline torch/onnx/symbolic_opset11.py:276]: Looks good. Do we have tests to cover both branches in this symbolic?

Comment (KsenijaS) [reply torch/onnx/symbolic_opset11.py:276]: yes, test_len covers the if branch, test_len_list covers the else branch

**Review by neginraoof — APPROVED** [review]
LGTM. Thanks.

**Review by shubhambhokare1 — APPROVED** [review]
LGTM

**Review by bzinodev — APPROVED** [review]

**Comment by KsenijaS** [comment]
@bzinodev Can we please merge this PR? Thanks!

**Pull request closed without merging**